Update SEO for more docs (#791)
hashicorp/terraform-docs-common: website/docs/cloud-docs/workspaces/state.mdx

---
page_title: Manage workspace state
description: >-

>   Workspaces have their own separate state data. Learn how state is used and how to access state from other workspaces.

Reviewer: ```suggestion
  Workspaces have their own separate state data. Learn how HCP Terraform uses state and how to access state from across workspaces.
```
